### PR TITLE
Feature/event form

### PR DESCRIPTION
**Pull request by kristenmb**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [x] Adds a new feature
- [ ] Improves an existing feature
- [ ] Fixes a bug
- [x] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
*Briefly, but clearly describe what you did, and explain any new code.*
- Implements initial form rendering
- Conditionally renders each section of the form as one section is completed
- updates general app styling slightly 

## *Instructions for reviewer:*
*Where should the reviewer start?*
- on the add-event page

*How should this be manually tested?*
- Go to add event page and click through form 

*Any background context you want to provide?*
- currently any button click will increment on the form counter and render the next part of the form. We will likely want to implement some error handling here before moving forward

*What are the relevant tickets?*
#8 

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

File: src/components/form/Form.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  background-color: #b3b3b3;
+  border-radius: .5rem;
+  height: 1.7rem;
+  width: 8rem;
+  margin-bottom: 1rem;
+}
+
+.form-button:hover {
+  background-color: #e2992f;
+}
```
Comment (DrewBradley) [inline src/components/form/Form.css:45]: I like this change. Are the buttons ever disabled? And if so do we have a style change in place.

File: src/components/form/Form.js [hunk 2]
```diff
@@ -1,19 +1,59 @@
...
+              type='text'
+              placeholder='State'
+              name='state'
+              value={state}
+              onChange={event => setState(event.target.value)}
+            />
+          </div>
+          <button
+            className='form-button'
+            onClick={event => incrementForm(event)}
```
Comment (DrewBradley) [inline src/components/form/Form.js:46]: Good job finding an effective way to render each page of the form!

**Review by DrewBradley — APPROVED** [review]
Looks great!

File: public/index.html [hunk 3]
```diff
@@ -2,6 +2,8 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap" rel="stylesheet">
```
Comment (richardltyler) [inline public/index.html:6]: I dig this font

File: src/components/form/Form.css (hunk 1 above)
Comment (richardltyler) [reply src/components/form/Form.css:45]: I second this. Good eye on stying details

File: src/components/form/Form.js [hunk 4]
```diff
@@ -1,19 +1,59 @@
...
+          >
+            Find Venues
+          </button>
+        </section>
+        { formCounter > 0 && (<FormVenue 
+          incrementForm={incrementForm}
+        />)}
+        { formCounter > 1 && <FormTime incrementForm={incrementForm}/> }
```
Comment (richardltyler) [inline src/components/form/Form.js:54]: This conditional rendering is 🔥 for the UI

File: src/components/form/FormTime.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
+import React, {useState} from 'react'
+import './Form.css';
+
+const FormTime = ({ date, setDate, time, setTime, allDayEvent, setAllDayEvent, eventName, setEventName, incrementForm}) => {
```
Comment (richardltyler) [inline src/components/form/FormTime.js:4]: This is kinda a future iteration thing, but do you think we can refactor to reuse one of these components?

**Review by richardltyler — APPROVED** [review]
Looks awesome!

**Pull request merged**